internal/convert.go

```diff
@@ -204,6 +205,27 @@ type Rule struct {
 	AddedOn           datetime.DateTime
 }
 
+// Stores information related to data transformations
+type Transformation struct {
+	Id                string
+	Name              string
+	Type              string
+	ObjectType        string
+	AssociatedObjects string
+	Enabled           bool
+	AddedOn           datetime.DateTime
+	Function          string
+	Input             []Input
+	Action            string
+	ActionConfig      map[string]interface{}
+}
+
+type Input struct {
+	Type     string
```

Comment:
What type is this ? Suggest you add sample values here to clarify what these fields are for